client/cluster_profile.go

```diff
@@ -207,6 +208,41 @@ func (h *V1Client) GetClusterProfile(uid string) (*models.V1ClusterProfile, erro
 	return resp.Payload, nil
 }
 
+type values struct {
+	HeaderMap map[string]string
+}
+
+func (v values) Get(key string) string {
```

Comment:
The `values` struct and custom headers implementation here are all duplicated from `apiutil/transport`. Please remove them.